outlook/src/classes/Company.ts

```diff
@@ -33,10 +42,27 @@ class Company {
         this.additionalInfo = {};
     }
 
-    static fromJSON(o: Object): Company {
+    static fromJSON(o: Object, enrichmentInfo?: EnrichmentInfo): Company {
         if (!o) return new Company();
         const company = Object.assign(new Company(), o);
         company.address = Address.fromJSON(o['address']);
+
+        if (enrichmentInfo != null) {
+            if (enrichmentInfo.type == EnrichmentInfoType.NoData) {
+                company.enrichmentStatus = EnrichmentStatus.enrichmentEmpty;
+            } else {
```

Comment:
Linting: elif ? Allow to remove an indentation and ease reading.

Reply:
Unfortunately there is no elif in TypeScript :/